src/components/ResultsBody/PaginatedList/VenueCard/index.jsx

```diff
@@ -0,0 +1,39 @@
+import styles from "./venueCard.module.css";
+import Typography from "@mui/material/Typography";
+
+export function VenueCard({ venue, backgroundUrl }) {
+  return (
+    <div className={styles.wrapper}>
+      <div
+        className={styles.pictureFrame}
+        style={{ "--background-url": `url(${backgroundUrl})` }}
+      >
+        `<div className={styles.heartBox}></div>
+        <div className={styles.nameBox}>
+          <Typography variant="boldOnCard">{venue.name}</Typography>
+        </div>
+        <div className={styles.arrowIcon}>
+          <i className={"fas fa-angle-right"}></i>
```

Comment:
you can use icons from mui-icons with material icons as explained here https://mui.com/material-ui/icons/